src/unix/stream.c

```diff
@@ -859,7 +859,16 @@ static void uv__write(uv_stream_t* stream) {
   }
 
   if (n < 0) {
-    if (errno != EAGAIN && errno != EWOULDBLOCK && errno != ENOBUFS) {
+        if (errno != EAGAIN && errno != EWOULDBLOCK && errno != ENOBUFS
+#if defined(__APPLE__)
```

Comment:
I'm not sure how other feel about this, but I'm not a huge fan of have an ifdef in the middle of a statement like this.